Tests/test_SearchIO_interpro_xml.py

```diff
@@ -0,0 +1,76 @@
+"""Tests for SearchIO InterproIO parsers."""
+
+import os
+import sys
+import unittest
+import warnings
+
+from Bio import BiopythonParserWarning
+from Bio import BiopythonExperimentalWarning
+
+
+with warnings.catch_warnings():
+    warnings.simplefilter('ignore', BiopythonExperimentalWarning)
+    from Bio.SearchIO import parse
+
+# test case files are in the Blast directory
+TEST_DIR = 'InterProScan'
+FMT = 'interpro-xml'
+
+
+def get_file(filename):
+    """Returns the path of a test file."""
+    return os.path.join(TEST_DIR, filename)
+
+
+class BlastnXmlCases(unittest.TestCase):
```

Comment:
This should be `InteproscanXmlCases`, I believe :).

Reply:
done